### PR TITLE
fixes to tracing k8s deployment page: Review with Arie & Yogev

### PR DESCRIPTION
**Pull request by YajB**
# What changed
https://deploy-preview-849--logz-docs.netlify.app/user-guide/distributed-tracing/k8s-deployment

- Adding comments and clear guidance within yaml to help users put in correct values. 
- added recommendation for creating Tracing token secret.
- remove commented text block 
- -

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/user-guide/distributed-tracing/k8s-deployment.md [hunk 1]
```diff
@@ -52,7 +54,7 @@ Deploy Jaeger agents and a collector - either the OpenTelemetry collector (recom
 
 
 1. Save the yaml below to a file and name it `config.yaml`.
-2. Edit the 2-letter region code if necessary (line 86).
+2. Edit the 2-letter region code, if needed. The region appears in the `apiVersion: v1 > data: > config.yaml > exporters: ` specification in the yaml code below).
```
Comment (yyyogev) [inline _source/user-guide/distributed-tracing/k8s-deployment.md:57]:
```suggestion
2. Edit the 2-letter region code, if needed. The region appears in the otel-collector-config ConfigMap under `config.yaml > exporters > logzio: ` specification in the yaml code below).
```

File: _source/user-guide/distributed-tracing/k8s-deployment.md [hunk 2]
```diff
@@ -89,11 +91,7 @@ spec:
       - image: otel/opentelemetry-collector-contrib:0.17.0
         name: otel-collector-logzio
         ports:
-          # - "1888:1888"   # pprof extension
-          # - "8888:8888"   # Prometheus metrics exposed by the collector
           # - "8889:8889"   # Prometheus exporter metrics
```
Comment (yyyogev) [inline _source/user-guide/distributed-tracing/k8s-deployment.md]:
```suggestion
```

Comment (yyyogev) [inline _source/user-guide/distributed-tracing/k8s-deployment.md]:
```suggestion
```

File: _source/user-guide/distributed-tracing/k8s-deployment.md [hunk 3]
```diff
@@ -246,7 +244,7 @@ spec:
 <div id="jaeger-collector">
 
 1. Save the yaml below to a file and name it `config.yaml`.
-2. Edit the 2-letter region code if necessary (line 44).
+2. Edit the 2-letter region code if needed. The region appears in the `apiVersion: apps/v1 spec: > template: > spec: > containers: > env: ` specification in the yaml code below.
```
Comment (yyyogev) [inline _source/user-guide/distributed-tracing/k8s-deployment.md:245]:
```suggestion
2. Edit the 2-letter region code if needed. The region appears in the  jaeger-collector-logzio deployment under `spec: > template: > spec: > containers: > env: ` specification in the yaml code below.
```

**Review by yyyogev — APPROVED** [review]
Great work!

File: _source/user-guide/distributed-tracing/k8s-deployment.md (hunk 2 above)
Comment (YajB) [reply _source/user-guide/distributed-tracing/k8s-deployment.md]: Removed in previous commit.

Comment (YajB) [reply _source/user-guide/distributed-tracing/k8s-deployment.md]: Removed in previous commit

File: _source/user-guide/distributed-tracing/k8s-deployment.md (hunk 1 above)
Comment (YajB) [reply _source/user-guide/distributed-tracing/k8s-deployment.md]: Committed the suggestion to modify line 57

**Pull request merged**